code/modules/jobs/job_types/roguetown/mercenaries/classes/desertrider.dm

```diff
@@ -70,8 +71,9 @@
 			H.change_stat("intelligence", 1)
 			H.change_stat("speed", 3)
 			backl = /obj/item/rogueweapon/sword/long/rider
-			shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/black
-			armor = /obj/item/clothing/suit/roguetown/armor/chainmail
+			shirt = /obj/item/clothing/suit/roguetown/armor/gambeson
+			armor = /obj/item/clothing/suit/roguetown/armor/leather/heavy/coat
+			pants = /obj/item/clothing/under/roguetown/trou/leather
```

Comment:
this variant is totally unarmored, they have armored leather pants that you can give them now